frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/GitMacheteGraphTable.java

```diff
@@ -117,7 +118,7 @@ public GitMacheteGraphTable(Project project) {
 
     addMouseListener(new GitMacheteGraphTableMouseAdapter());
 
-    subscribeToVcsRootChanges();
+    subscribeToVcsRepositoryChange();
```

Comment:
Whoops... that's getting confusing :/

Reply:
change/changes, vcs/git... but in fact the two methods refer to different things

Reply:
Sth along the lines of `selection change` vs `repository content/files change`?